### PR TITLE
Fixes the kitchen spike turning people upside down forever

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

**Comment by Shadowlight213** [comment]
Fun removal

**Comment by Ordonis** [comment]
Why would you do this to us, we were just starting to unlock the true power of handstands

**Comment by PKPenguin321** [comment]
how many times has this been fixed

**Comment by imsxz** [comment]
Getting teleported off (via bluespace crystal, swarmer, etc) also causes you to remain upside down, dunno if this covers that.

**Comment by awesine** [comment]
feature

File: code/game/objects/structures/kitchen_spike.dm [hunk 1]
```diff
@@ -129,15 +129,24 @@
...
+	src.visible_message(text("<span class='danger'>[M] falls free of [src]!</span>"))
+	unbuckle_mob(M,force=1)
+	M.emote("scream")
+	M.AdjustKnockdown(20)
+
+/obj/structure/kitchenspike/Destroy()
+	if(has_buckled_mobs())
+		for(var/mob/living/L in buckled_mobs)
+			release_mob(L)
+	..()
```
Comment (lzimann) [inline code/game/objects/structures/kitchen_spike.dm]: `return ..()`

**Pull request merged**